client.go

```diff
@@ -496,17 +495,7 @@ func (c *Client) SetTLSClientConfig(config *tls.Config) *Client {
 //		resty.SetTimeout(time.Duration(1 * time.Minute))
 //
 func (c *Client) SetTimeout(timeout time.Duration) *Client {
```

Comment:
Currently `resty` support Go1.2 and above. HTTP Client `SetTimeout` was introduced in Go1.3. You see [travis build is failed for Go1.2](https://travis-ci.org/go-resty/resty/jobs/209669843)

Can you please handle these support with build constraints?